### DISCUSSION TITLE
Will the library reduce overall application throughput (as did use parallel)?

### DISCUSSION THREAD

**Discussion by buzz100** [SixLabors/ImageSharp.Web#252] [Q&A]
Hi,
Their was an old post which raised some concern about this library, https://photosauce.net/blog/post/5-reasons-you-should-stop-using-systemdrawing-from-aspnet, It said it had done a really good job but states

"One word of warning with this library, though, since we’re talking about server apps: As of now, its default configuration uses Parallel. For internally to speed up some of its operations, which means it will tie up more worker threads from your ASP.NET thread pool, ultimately reducing overall application throughput."

Is this still the case and if so is the impact anything to be concerned about with ImageSharp.Web V2?

**Comment by JimBobSquarePants** [comment]
No, that's pretty outdated.
